cmd/hatchet-migrate/migrate/migrations/20251224145920_v1_0_65.sql

```diff
@@ -0,0 +1,78 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TYPE v1_event_type_olap_new AS ENUM (
+    'RETRYING',
+    'REASSIGNED',
+    'RETRIED_BY_USER',
+    'CREATED',
+    'QUEUED',
+    'REQUEUED_NO_WORKER',
+    'REQUEUED_RATE_LIMIT',
+    'ASSIGNED',
+    'ACKNOWLEDGED',
+    'SENT_TO_WORKER',
+    'SLOT_RELEASED',
+    'STARTED',
+    'TIMEOUT_REFRESHED',
+    'SCHEDULING_TIMED_OUT',
+    'FINISHED',
+    'FAILED',
+    'CANCELLED',
+    'TIMED_OUT',
+    'RATE_LIMIT_ERROR',
+    'SKIPPED',
+    'COULD_NOT_SEND_TO_WORKER'
+);
+
+ALTER TABLE v1_task_events_olap_tmp
+    ALTER COLUMN event_type TYPE v1_event_type_olap_new
+    USING event_type::text::v1_event_type_olap_new;
+
+ALTER TABLE v1_task_events_olap
+    ALTER COLUMN event_type TYPE v1_event_type_olap_new
+    USING event_type::text::v1_event_type_olap_new;
+
```

Comment:
any idea if these will lock these tables and perform a sequential scan to change the types? I didn't see anything on that in the docs on a cursory glance, but if it does that might be a non-starter